applications/feedback_sentiment_analyzer/README.md

```diff
@@ -0,0 +1,42 @@
+# Feedback Sentiment Analyzer (FSA)
+
+Feedback Sentiment Analyzer (FSA) is an example application that showcases AWS services and SDKs.
+
+Specifically, it solves a ficticious use case of a hotel in Paris, France, which receives feedback from guests in a variety of foreign languages.
+
+## SDKs this application is implemented in
```

Comment:
```suggestion
## Lambda implementations

The business logic of the app is implemented in Lambda functions that are available in the following AWS SDKS:
```

Reply:
I don't think this is clear enough for the customer to realize what we're trying to communicate. 

How about `SDK implementations`?

Reply:
That implies that it's an implementation _of_ the SDK not _with_ the SDK, but we can roll with whatever you've got now.